common/models/access-token.js

```diff
@@ -65,20 +65,23 @@ module.exports = function(AccessToken) {
   /*!
    * Hook to create accessToken id.
    */
+  AccessToken.observe('before save', function(ctx, next) {
+    if (!ctx.instance) {
+      if (ctx.data.id) return next();
 
-  AccessToken.beforeCreate = function(next, data) {
-    data = data || {};
+      // Example: AccessToken.updateAll()
+      var msg = 'AccessToken does not support data-based updates.';
```

Comment:
This message is confusing... I'm not even sure what it is trying to say. It must be updated with `inst.save()` and counterparts? And not `updateAttributes(data)`?